lib/view/weekView.dart

```diff
@@ -40,45 +154,87 @@ class WeekView extends StatelessWidget {
 
 //Each of these navigates to dayView when tapped
 class MultiDayCard extends StatefulWidget {
-  const MultiDayCard(this.index, {super.key});
+  const MultiDayCard(this.index, this.startDate, {super.key});
   final int index;
+  final DateTime startDate;
 
   @override
-  State<StatefulWidget> createState() => _MultiDayCardState(index);
+  State<StatefulWidget> createState() => _MultiDayCardState(index, startDate);
 }
 
 class _MultiDayCardState extends State<MultiDayCard> {
   int eventCount = 0;
-  int taskCount = 0;
   List<Event> eventsToday = [];
-  List<Task> tasksDueToday = [];
-  _MultiDayCardState(this.index) {
-    DateTime date = DateTime(DateTime.now().year, DateTime.now().month, DateTime.now().day).add(Duration(days: index));
+  int index;
+  DateTime startDate;
+  _MultiDayCardState(this.index, this.startDate) {
+    db
+        .getListOfEventsInDay(date: getDateOnly(startDate, offsetDays: index))
+        .then((value) => setState(() {
+              eventCount = value.length;
+              eventsToday = value;
+            }));
+  }
 
-    db.getListOfEventsInDay(date: getDateOnly(DateTime.now(), offsetDays:index)).then((value) => setState(() {
-          eventCount = value.length;
-          eventsToday = value;
-        }));
-    db.getTasksDueDay(DateTime.now()).then((value) => setState(() {
-          taskCount = value.length;
-          tasksDueToday = value;
-        }));
+  String weekdayToString(int weekday) {
+    switch (weekday) {
+      case 1:
+        return "Mon";
+      case 2:
+        return "Tue";
+      case 3:
+        return "Wed";
+      case 4:
+        return "Thu";
+      case 5:
+        return "Fri";
+      case 6:
+        return "Sat";
+      case 7:
+        return "Sun";
+      default:
+        return "";
+    }
```

Comment:
We can use this
![image](https://github.com/Scrumbags115/plannertarium/assets/130195350/116320cf-b2b3-48d4-bb46-6a5ca7a82e69)